tools/delete_imaging_upload.pl

```diff
@@ -187,34 +187,34 @@ =head2 Methods
     DELETE_PROTOCOLS          => DEFAULT_DELETE_PROTOCOLS,
     DELETE_MRI_PARAMETER_FORM => DEFAULT_DELETE_MRI_PARAMETER_FORM,
     KEEP_DEFACED              => DEFAULT_KEEP_DEFACED,
-    BACKUP_BASENAME           => '',
+    BACKUP_PATH               => '',
     UPLOAD_ID                 => ''
 );
 
 my $scanTypeList           = undef;
 
 my @opt_table = (
-    ['-profile'  , 'string'       , 1, \$options{'PROFILE'}, 
+    ['-profile'    , 'string'       , 1, \$options{'PROFILE'}, 
      'Name of config file in ../dicom-archive/.loris_mri (defaults to "prod")'],
-    ['-backup'   , 'string'       , 1, \$options{'BACKUP_BASENAME'}, 
-     'Basename of the backup file (defaults to "imaging_upload_backup")'],
-    ['-ignore'   , 'const'        , 0, \$options{'DIE_ON_FILE_ERROR'}, 
+    ['-backup_path', 'string'       , 1, \$options{'BACKUP_PATH'}, 
+     'Path of the backup file (defaults to "imaging_upload_backup")'],
```

Comment:
maybe modify to "defaults to "imaging_upload_backup" **in the working directory**" ? Is that true?

Reply:
Good idea. Done.